gflownet/utils/buffer.py

```diff
@@ -21,14 +22,13 @@ def __init__(
         env,
         proxy,
         replay_capacity=0,
-        output_csv=None,
         data_path=None,
         train=None,
         test=None,
         logger=None,
         **kwargs,
     ):
-        self.logger = logger
+        self.datadir = logger.datadir
```

Comment:
`logger` defaults to `None`, if this is allowed it should be handled here; otherwise make it required?

Reply:
Good point, done here: https://github.com/alexhernandezgarcia/gflownet/pull/322/commits/f8c235ce8d9d3a15b6b22646ccd40ab61d58010a